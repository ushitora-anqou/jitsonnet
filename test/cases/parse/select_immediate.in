{y: {z: "a"}}.y.z
