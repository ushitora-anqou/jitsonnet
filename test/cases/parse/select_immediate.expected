{ Syntax.v =
  (Syntax.Select
     ({ Syntax.v =
        (Syntax.Select
           ({ Syntax.v =
              (Syntax.Object
                 (Syntax.ObjectMemberList
                    [(Syntax.MemberField
                        (Syntax.Field
                           ((Syntax.FieldnameID
                               { Syntax.v = "y";
                                 loc =
                                 { Syntax.fname =
                                   "test/cases/parse/select_immediate.in";
                                   ran =
                                   (Some { Syntax.startpos =
                                           { Syntax.line = 1; column = 1 };
                                           endpos =
                                           { Syntax.line = 1; column = 2 } })
                                   }
                                 }),
                            false, (Syntax.H 1),
                            { Syntax.v =
                              (Syntax.Object
                                 (Syntax.ObjectMemberList
                                    [(Syntax.MemberField
                                        (Syntax.Field
                                           ((Syntax.FieldnameID
                                               { Syntax.v = "z";
                                                 loc =
                                                 { Syntax.fname =
                                                   "test/cases/parse/select_immediate.in";
                                                   ran =
                                                   (Some { Syntax.startpos =
                                                           { Syntax.line = 1;
                                                             column = 5 };
                                                           endpos =
                                                           { Syntax.line = 1;
                                                             column = 6 }
                                                           })
                                                   }
                                                 }),
                                            false, (Syntax.H 1),
                                            { Syntax.v = (Syntax.String "a");
                                              loc =
                                              { Syntax.fname =
                                                "test/cases/parse/select_immediate.in";
                                                ran =
                                                (Some { Syntax.startpos =
                                                        { Syntax.line = 1;
                                                          column = 8 };
                                                        endpos =
                                                        { Syntax.line = 1;
                                                          column = 11 }
                                                        })
                                                }
                                              })))
                                      ]));
                              loc =
                              { Syntax.fname =
                                "test/cases/parse/select_immediate.in";
                                ran =
                                (Some { Syntax.startpos =
                                        { Syntax.line = 1; column = 4 };
                                        endpos =
                                        { Syntax.line = 1; column = 12 } })
                                }
                              })))
                      ]));
              loc =
              { Syntax.fname = "test/cases/parse/select_immediate.in";
                ran =
                (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
                        endpos = { Syntax.line = 1; column = 13 } })
                }
              },
            "y"));
        loc =
        { Syntax.fname = "test/cases/parse/select_immediate.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
                  endpos = { Syntax.line = 1; column = 15 } })
          }
        },
      "z"));
  loc =
  { Syntax.fname = "test/cases/parse/select_immediate.in";
    ran =
    (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
            endpos = { Syntax.line = 1; column = 17 } })
    }
  }